starlark/testdata/math.star

```diff
@@ -17,16 +17,25 @@ assert.eq(math.ceil(10.0), 10.0)
 assert.eq(math.ceil(0), 0.0)
 assert.eq(math.ceil(1), 1.0)
 assert.eq(math.ceil(10), 10.0)
-assert.eq(math.ceil(inf), inf)
-assert.eq(math.ceil(nan), nan)
 assert.eq(math.ceil(-0.0), 0.0)
 assert.eq(math.ceil(-0.4), 0.0)
 assert.eq(math.ceil(-0.5), 0.0)
 assert.eq(math.ceil(-1.0), -1.0)
 assert.eq(math.ceil(-10.0), -10.0)
 assert.eq(math.ceil(-1), -1.0)
 assert.eq(math.ceil(-10), -10.0)
-assert.eq(math.ceil(-inf), -inf)
+assert.eq(type(math.ceil(0)), "int")
+assert.eq(type(math.ceil(0.4)), "int")
+assert.eq(type(math.ceil(10)), "int")
+assert.eq(type(math.ceil(-10.0)), "int")
+assert.eq(type(math.ceil(-0.5)), "int")
+assert.eq(math.ceil(1.8446744073709552e+19), 1.8446744073709552e+19)
```

Comment:
It's unclear what the first constant signifies. `float(1<<64)`? The same thing plus or minus one? Derive the argument to make it clear. 

Is the second argument intentionally a float? It may compare equal to the int returned by ceil, but it would be better to use an expression that is actually an int.